core/src/main/java/org/kiwix/kiwixmobile/core/search/viewmodel/Action.kt

```diff
@@ -30,6 +30,7 @@ sealed class Action {
   data class OnItemClick(val searchListItem: SearchListItem) : Action()
   data class OnItemLongClick(val searchListItem: SearchListItem) : Action()
   data class Filter(val term: String) : Action()
+  data class ScreenOrigin(val searchOrigin: SearchOrigin) : Action()
```

Comment:
We can come up with something better for this name `ScreenWasStartedFrom`, an action should answer the question "what happened?"